### PR TITLE
add operator status check func

### PR DESCRIPTION
**Pull request by LeslieKuo**
Installation scripts enhancements
Issue:https://github.com/IBM/community-automation/issues/67

### CODE REVIEW

**Review by wkrapohl — COMMENTED** [review]
I see your just adding two functions, that are not being used anyware. 
Is this what you want to do. Not sure we should just add code that is not used.

**Comment by LeslieKuo** [comment]
@wkrapohl update the scripts and need review. Thanks

File: ansible/roles/request-ocs/files/ocs_install.sh [hunk 1]
```diff
@@ -152,26 +175,52 @@ spec:
...
+#   echo "Waiting for storage classes to come up: $(date)"
+#   sleep 10
+#   ((scCounter ++))
+#   if [ $scCounter -eq 60 ]; then
+#    exit 1
+#   fi
+#   oc get sc --no-headers | cut -f1 -d' ' | grep noobaa >/dev/null
+#   rc=$?
+# done
+# oc get sc
```
Comment (wkrapohl) [inline ansible/roles/request-ocs/files/ocs_install.sh:193]: Need to keep the code that set the default storageclass based on setDefault=True lines 193 to 200,

Your code should only replace 178 to 192

Comment (wkrapohl) [reply ansible/roles/request-ocs/files/ocs_install.sh:193]: I'm not sure what is exactly wrong ... have not had the time to investigate... but have not been able to get these changes to work on our VMware setup.

Comment (rayashworth) [reply ansible/roles/request-ocs/files/ocs_install.sh:193]: @LeslieKuo have you had time to look over walts comments?

**Comment by rayashworth** [comment]
marking this as wont fix for now, we can revisit if @LeslieKuo has a need.

**Pull request closed without merging**